python/ray/tune/integration/wandb.py

```diff
@@ -254,6 +254,8 @@ class WandbLogger(Logger):
     def _init(self):
         config = self.config.copy()
 
+        config.pop("callbacks", None)  # Remove callbacks
```

Comment:
is this safe? should we do like `log_once` just to inform the user that this is rllib specific?

Reply:
I mean it's just for the (wandb) logger and we copy the dict right before it. It would be a problem if someone passed their own config parameters with `callback` as the parameter name. Though I'm not sure if he wouldn't run into other problems then anyway?